implementations/c/ockam/log/log.c

```diff
@@ -39,7 +39,7 @@ static ockam_log_function_t ockam_log_function = ockam_log_printf;
 static ockam_log_function_t ockam_log_function = NULL;
 #endif
 
-static ockam_log_level_t ockam_log_level = OCKAM_LOG_LEVEL_DEBUG;
+static ockam_log_level_t ockam_log_level = OCKAM_LOG_LEVEL_INFO;
```

Comment:
Is this the default level? If so maybe specify it as another variable and assign the default variable to `ockam_log_level` for readability

Reply:
@mikelodder7 added comment. Works for you?